### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by brianwchou**
## Description
- update ganache-time-traveler module
- create a descriptive readme describing project

### CODE REVIEW

**Comment by brianwchou** [comment]
Is there a clean way to download the developer dependencies at the Truffle project level? When I do a fresh clone, I want to use `npm install` to download everything but it downloads too many files. There's also a conflict with webpack versions when doing it this way.

**Comment by ejwessel** [comment]
Not sure what you mean by it downloads too many files. What command are you running??

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Hangman
+Play hangman on the Ethereum network. Hangman is a word guessing game the word completely or guess letters the word contains. Each try uses an attempt and players win when guessed correctly and lose when all attempts have been used. Random word is fetched from the [Wikipedia](Wikipedia.org) API via Chainlink
```
Comment (ejwessel) [inline README.md]: Something like the following:

```
Play hangman on the Ethereum network

Hangman is a game where a player must guess the word in the allotted guesses. 
Players are allowed to guess letters and words. 
Guessing a word or letter incorrectly uses an attempt. 
When all attempts are used or if the word is identified the game ends.
```  

Take a look at what Wikipedia says. 

Also there should be another section that talks about how it works on a tech level. It doesn't need to be in first paragraph.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+### Local Setup
+1. clone the hangman repository
```
Comment (ejwessel) [inline README.md]: Instead of saying clone. Give the command too.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+2. cd into the repo
+3. ``
```
Comment (ejwessel) [inline README.md]: Provide the rest of the commands when you're ready.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+
+### Tech Framework
```
Comment (ejwessel) [inline README.md]: Look at the dependencies

File: package.json [hunk 5]
```diff
@@ -33,7 +34,7 @@
   "dependencies": {
     "@gnosis.pm/mock-contract": "2.0.4",
     "chainlink": "^0.6.4",
-    "ganache-time-traveler": "github:ejwessel/GanacheTimeTraveler",
+    "ganache-time-traveler": "^1.0.0",
```
Comment (ejwessel) [inline package.json]: Thanks for doing this.

File: package.json [hunk 6]
```diff
@@ -11,8 +11,9 @@
     "truffle-compile": "truffle compile",
     "truffle-test": "yarn clean; yarn truffle-migrate && truffle test",
     "ganache": "ganache-cli -d",
-    "web:start": "cd ./web && npm start",
-    "web:test": "cd ./web && npm test",
+    "web:start": "cd ./web; yarn start",
+    "web:test": "cd ./web; yarn test",
+    "web:install": "cd ./web; yarn",
```
Comment (ejwessel) [inline package.json]: Running `yarn` installs everything. We should just make sure this is apart of the setup steps

Comment (ejwessel) [reply package.json]: Oh I guess this is for web. NVM makes sense.

**Comment by ejwessel** [comment]
You should be running yarn and not npm

**Comment by brianwchou** [comment]
> Not sure what you mean by it downloads too many files. What command are you running??
So I learned that you cannot run `yarn` at the root level of the project because it downloads a large set of dependencies that include an older version of webpack. This conflicts with running the react application and prevents you from launching the client locally. I spent yesterday trying to figure out if theres a method of getting around this but i found nothing so far. Sounds like if i want to download the zepplin solidity contracts and ganache-time-traveller I need to install them separately instead of using the convenience of just `yarn`

**Comment by brianwchou** [comment]
As discovered trufflecon, the mass dependencies were from the package.json file re-downloading truffle. The extra webpack dependencies are from the chainlink package.

**Comment by brianwchou** [comment]
Techincal description and local set up still need to be finished.

File: README.md [hunk 7]
```diff
@@ -1,20 +1,21 @@
 ## Hangman
 Play hangman on the Ethereum network. 
 
-Hangman is a game where a player has limited attempts to guess a secret word. The player is allowed to guess a letter contained in secret or guess the entire secret. Each incorrect guess uses an attempt. When all attemps have been made or if the word has been guessed correctly the game ends.
+Hangman is a game where a player has limited attempts to guess a secret word. The player is allowed to guess a letter contained in secret or guess the entire secret. Each incorrect guess uses an attempt. When all attempts have been made or if the word has been guessed correctly the game ends.
```
Comment (ejwessel) [inline README.md]: contained in secret -> contained in the secret

**Comment by ejwessel** [comment]
I think this is ready at this point. Anything else you want to add we can add after.

**Pull request merged**